apps/sms/views/conversation/test/unit/link_helper_test.js

```diff
@@ -153,12 +153,14 @@ suite('link_helper_test.js', function() {
       test('non ascii prefix', function() {
         testURLMatch('àmozilla.org', 'mozilla.org', true);
       });
+      test('Bug 941763', function() {
```

Comment:
IMO bug number is not important to write in test description. Instead, give better/detail description :)